### PR TITLE
docs: Add extra information for GitLab reporter's integration with GitLab Code Quality

### PR DESCRIPTION
**Pull request by 0x346e3730**

### CODE REVIEW

File: doc/usage.rst [hunk 1]
```diff
@@ -211,6 +211,11 @@ Then, add the following command to your CI:
 
 Where ``$COMMIT_RANGE`` is your range of commits, e.g. ``$TRAVIS_COMMIT_RANGE`` or ``HEAD~..HEAD``.
 
+### GitLab Code Quality Integration
+
+If you want to integrate with GitLab's Code Quality feature you will need to use both ``--format=gitlab`` and ``--diff``
```
Comment (ostrolucky) [inline doc/usage.rst:216]:
```suggestion
If you want to integrate with GitLab's Code Quality feature, in order for report to contain correct line numbers, you will need to use both ``--format=gitlab`` and ``--diff``
```

Comment (0x346e3730) [reply doc/usage.rst]: Done

File: doc/usage.rst [hunk 2]
```diff
@@ -211,6 +211,11 @@ Then, add the following command to your CI:
 
 Where ``$COMMIT_RANGE`` is your range of commits, e.g. ``$TRAVIS_COMMIT_RANGE`` or ``HEAD~..HEAD``.
 
+### GitLab Code Quality Integration
```
Comment (Wirone) [inline doc/usage.rst:214]: This is not Markdown, but Restructured Text, so this style of heading does not work. According to [the docs](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections) it must be "underlined" with different character than in previous headings (`=` and `-` already used), so another nesting level is determined automatically, like:

```suggestion
GitLab Code Quality Integration
###############################
```

Comment (0x346e3730) [reply doc/usage.rst]: Fixed

**Review by ostrolucky — APPROVED** [review]

**Review by Wirone — APPROVED** [review]

**Comment by Wirone** [comment]
Thank you @0x346e3730 and welcome to the contributors group 🍻🙂.

**Comment by 0x346e3730** [comment]
Thank you guys 😃

**Pull request merged**